src/app/components/course/course-registration/course-register.component.ts

```diff
@@ -7,10 +7,10 @@ import {MessageService} from '@app/_services/message.service';
 import {CourseRegistrationResponse, REGISTRATION_STATUS, MESSAGE_TYPES} from '@app/_models';
 
 const STEPPER_STAGES = {
-    ENTER_NAME: 0,
-    CONFIRM_IDENTITY: 1,
-    VERIFICATION: 2,
-    REGISTERED: 3,
+    enterName: 0,
```

Comment:
This was correct to be PascalCase I'll modify the eslint. Please pull from master and change this back.

Reply:
UPPER_CASE*